src/board/board.mjs

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable no-use-before-define */
+import { checkForWin } from './game';
+
```

Comment:
i think you could remove the extra lines between the import statements, usually a new line after an import statements indicates to developers that the imports have stopped and the code has started

Reply:
✔️